shopware/administration/6.6/bin/build-administration.sh

```diff
@@ -85,3 +85,4 @@ fi
 
 (cd "${ADMIN_ROOT}"/Resources/app/administration && npm run build)
 [[ ${SHOPWARE_SKIP_ASSET_COPY-""} ]] ||"${BIN_TOOL}" assets:install
+[[ ${SHOPWARE_SKIP_CACHE_CLEAR-""} ]] || "${BIN_TOOL}" cache:clear --active-only
```

Comment:
i don't think cache-clear have active only 🤔

Reply:
Ohh, had no testing machine, gonna change that real quick